cost-analyzer/values.yaml

```diff
@@ -2438,6 +2459,8 @@ networkCosts:
   additionalLabels: {}
   nodeSelector: {}
   annotations: {}
+  # Annotations to be added to network cost daemonset template
+  daemonsetAnnotations: {}
```

Comment:
I'm not a big fan of using configs like `daemonsetAnnotations` and `deploymentAnnotations`. Could we stick to only using `annotations` and `podAnnotations` for consistency across the chart?

Doing this may be a breaking change for a few templates (network costs, awsstore), but I think it will be ok. We can call out in the release notes.

Reply:
sure, will update them. Had named these so that they might become a breaking change. But if we can specify it in the release notes, doing it consistently will be better.

Reply:
@jessegoodier @chipzoller Do you agree with this approach? Are you also ok with making a minor breaking change here?

Reply:
I think renaming two of these is ok if called out in release notes. I suspect they're only lightly used.

Reply:
Ok, thanks for input! @mittal-ishaan Before approving this PR, could you add comments to this PR's description listing which templates will have breaking changes?